src/app/App.tsx

```diff
@@ -1,18 +1,48 @@
-import React, { useState } from 'react'
+import React, { useContext } from 'react'
 import '../assets/scss/_index.scss'
 import { version } from '../../package.json'
 import LoginScreen from './LoginScreen'
 import DashboardScreen from './DashboardScreen'
 import RifFooter from '../components/RifFooter/RifFooter'
+import { Web3ProviderContext } from '../providerContext'
+import RLogin from 'jesse-rlogin'
+import WalletConnectProvider from '@walletconnect/web3-provider'
 
 const App = () => {
-  const [isLoggedIn, setIsLoggedIn] = useState<boolean>(false)
+  const context = useContext(Web3ProviderContext)
 
+  const handleLogin = () => {
+    const rLogin = new RLogin({
```

Comment:
We can construct `RLogin` outside the component. Then, `handleLogin` would send `connect`.

Remember that the constructor is doing `createChild` and `render` into that child. The result of opening the modal, closing it and opening it again is

<img width="1092" alt="Screen Shot 2020-11-10 at 01 29 51" src="https://user-images.githubusercontent.com/36084092/98628049-6104c800-22f4-11eb-9a8d-844ffc931b97.png">

That is also breaking the click handler I don't know exactly why

Reply:
@ilanolkies Yes, I tried elevating the init of rLogin in App and passing it down from props into Signup. However, when I do this, the "metamask" button is no longer clickable. I'll take a look at this today, as I think it is combined with the non-clickable button when the modal is closed and reopened, as discussed yesterday.

Reply:
I updated this code to move `rLogin` outside of `App.tsx`. However, because context is meant to work with components, setting the context needs to be handled inside of `App.tsx`. I think context might be the wrong approach.